src/category_theory/monoidal/category.lean

```diff
@@ -0,0 +1,405 @@
+-- Copyright (c) 2018 Michael Jendrusch. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Michael Jendrusch, Scott Morrison
+import category_theory.monoidal.tensor_product
+import category_theory.natural_isomorphism
+import tactic -- TODO switch this back to tactic.basic once relevant PRs land.
+import tactic.slice
+
+open category_theory
+open tactic
+
+universes v u
+
+open category_theory
+open category_theory.category
+open category_theory.iso
+open category_theory.monoidal
+
+namespace category_theory.monoidal
+class monoidal_category (C : Sort u) extends category.{v} C :=
+-- curried tensor product of objects:
+(tensor_obj               : C → C → C)
+-- curried tensor product of morphisms:
+(tensor_hom               :
+  Π {X₁ Y₁ X₂ Y₂ : C}, hom X₁ Y₁ → hom X₂ Y₂ → hom (tensor_obj X₁ X₂) (tensor_obj Y₁ Y₂))
+-- tensor product laws:
+(tensor_map_id'           :
```

Comment:
Maybe we can just call this `tensor_id'`? Or am I missing something?

Reply:
Renamed.